schema/changelog-3.15.xml

```diff
@@ -83,5 +83,54 @@
       <column name="limitcommands" type="BOOLEAN" defaultValueBoolean="false" />
     </addColumn>
 
+    <addColumn tableName="notifications">
+      <column name="allways" type="BOOLEAN" defaultValueBoolean="false" valueBoolean="true">
```

Comment:
Make old notifications "always" to make migration transparent to users.

Reply:
There is typo in `always`. There should be only one `L`.